pkgs/tools/filesystems/gocryptfs/default.nix

```diff
@@ -2,7 +2,9 @@
 { stdenv, buildGoPackage, fetchFromGitHub, openssl, pandoc, pkgconfig }:
 
 let
-  version = "v1.6.1";
+  pname = "gocryptfs";
```

Comment:
this should be below in `buildGoPackage rec` and remove `name`.

Reply:
Resolved with 2e793c1e6e86ffb8bcd08ee1309caf3b95f8f740